### PR TITLE
Add command line interface using thor

### PR DESCRIPTION
**Pull request by gravesm**
I would like to be able to run this as a command line tool outside of any specific application. If I'm alone in this, I'm happy to maintain a fork, but I figured I would see if there was any interest.

### CODE REVIEW

**Comment by mejackreed** [comment]
:+1: great idea @gravesm ... Could we just not bump the version until we do a `rake release`?

File: lib/geo_combine/cli.rb [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+module GeoCombine
+  class CLI < Thor
+    desc 'clone', 'Clone all OpenGeoMetadata repositories'
+    def clone
+      Rake::Task['geocombine:clone'].invoke
+    end
+
+    desc 'pull', '"git pull" OpenGeoMetadata repositories'
+    def pull
+      Rake::Task['geocombine:pull']
```
Comment (mejackreed) [inline lib/geo_combine/cli.rb]: Should there be an `.invoke` here?

File: README.md [hunk 2]
```diff
@@ -64,7 +66,7 @@ Runs `git pull origin master` on all cloned repositories in `./tmp/opengeometada
 ### Index all of the GeoBlacklight documents
 
 ```sh
-$ rake geocombine:index
+$ bundle exec geocombine index
 ```
```
Comment (mejackreed) [inline README.md]: Could we also specify here that the tasks are also available via rake?

File: lib/geo_combine/cli.rb (hunk 1 above)
Comment (gravesm) [inline lib/geo_combine/cli.rb]: Yeah, there also appears to be a problem with loading the rake tasks if you're not in the gem's directory. Fixes incoming.

**Comment by mejackreed** [comment]
Awesome sauce! :pizza:

**Pull request merged**